### PR TITLE
Portfolio Contact Page Project

### PR DESCRIPTION
**Pull request by Htetaungkyaw71**
This is Contact-page project for pair programming. Detail below

I added some Stylesheet style.css
I added some Markup index.html
I added gitignore file
I added descriptive readme file
I added stylelint for linters
I added form-validation in html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -215,5 +215,23 @@ <h1 class="name">I'm Ifeanyi, Glad to see you!</h1>
...
+          <form action="https://formspree.io/f/xzbwrozq" method="POST">
+            <input type="text" placeholder="Enter your fullname" name="name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="input you email" required>
+            <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message" maxlength="500" required></textarea>
+            <button type="submit" class="submit">Get in touch</button>
+          </form>
+        </div>
+      </div>
+    </section>
+  </footer>
```
Comment (HAMMAS-SALEEM) [inline index.html:235]: white-spacing-contact-form

- [ ] Your contact form is working as expected :+1: But it is not responsive on all mobile screens as you can see in the image above :arrow_up: in the image :+1:

:thinking: **Hint:** `Kindly remove width from body selector in the css file` :blush:

- [ ] On refreshing the page, your form should be filled with the data inside your localStorage. Kindly implement this feature :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:

## Hey Team (@ifeanyi305, @Htetaungkyaw71) :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following requirements
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_See Below Comments_

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @ifeanyi305, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PRR ✔️ 
- [ ] Used correct Github flow ✔️ 

## Optional suggestions

- [OPTIONAL] The commit message should be written in the imperative tense and capitalized, please remember this in your next commits 🙃 😉 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — COMMENTED** [review]
Hi @Htetaungkyaw71 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that while this is a pair programming project, You are expected to implement the above requirements in all repositories in your pair-programming group. That means that each group member needs to open a new pull request in their repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html (hunk 1 above)
Comment (ifeanyi305) [reply index.html:235]: Dear @HAMMAS-SALEEM,
Thank you for taking the time to review my code and for providing feedback. I appreciate your positive feedback on the contact form and your suggestion for improving its responsiveness. Please find my response below:

- Your contact form is working as expected 👍 But it is not responsive on all mobile screens as you can see in the image above ⬆️ in the image 👍
- 🤔 Hint: Kindly remove width from body selector in the css file 😊

Thank you for bringing to my attention the issue with the responsiveness of the contact form on mobile screens. I appreciate the hint you provided to remove the width from the body selector in the CSS file. I will make the necessary changes to ensure that the form is responsive on all mobile screens.

- On refreshing the page, your form should be filled with the data inside your localStorage. Kindly implement this feature 😊

Thank you for reminding me of the feature to fill the form with the data inside the localStorage on refreshing the page. I will implement this feature to improve the functionality and usability of the contact form.

Once again, thank you for your feedback and guidance. I will make the necessary changes and resubmit the code for your review.

**Pull request merged**